test/mongodb.test.js

```diff
@@ -4,6 +4,36 @@ var should = require('./init.js');
 
 var Superhero, User, Post, PostWithStringId, db;
 
+describe('lazyConnect', function() {
+  it('should skip connect phase (lazyConnect = true)', function(done) {
+    var ds = getDataSource({
+      host: '127.0.0.1',
+      port: 4,
+      lazyConnect: true,
+    });
+    var errTimeout = setTimeout(function() {
+      done();
+    }, 5000);
```

Comment:
I remember see 5000 in one test case somewhere else (but could not remember where it is:( ).
On my local machine, the passing time is always 1000+ ms, so my suggestion is double it, which is 2000ms, to guarantee enough time to trigger the error.